.gitignore

```diff
@@ -33,3 +33,4 @@ releasenotes
 *~
 *.tmp
 .DS_Store
+.idea/
```

Comment:
Line feed missing.

Reply:
Removed the row from the file since it was more related to my setup I configured it on my side instead